app/models/spree/apple_pay_payment_decorator.rb

```diff
@@ -1,9 +1,9 @@
-module SpreeGateway
+module Spree
   module ApplePayPaymentDecorator
     def apple_pay?
       payment_method.is_a? Spree::Gateway::StripeApplePayGateway
     end
   end
 end
 
-Spree::Payment.prepend SpreeGateway::ApplePayPaymentDecorator
+Spree::Payment.prepend Spree::ApplePayPaymentDecorator
```

Comment:
why do want to move it to `Spree` namespace?

Reply:
@chhga that's Zeitwerk requirement, we would need to move this file into `spree_gateway` directory to have the `SpreeGateway` namespace working, but that, on the other hand, would break decorator loading